### PR TITLE
Some HTTP to HTTPS.

### PR DESCRIPTION
**Pull request by JulienPalard**
While shaving a yak I though it was a good idea to proofread some old links.

### CODE REVIEW

File: users/tests/test_models.py [hunk 1]
```diff
@@ -60,12 +60,12 @@ def test_create_usergroup(self):
         group = UserGroup.objects.create(
             name='PLUG',
             location='London, UK',
-            url='http://meetup.com/plug',
+            url='https://meetup.com/plug',
```
Comment (berkerpeksag) [inline users/tests/test_models.py]: This is just a test, no need to change.

File: templates/python/inner.html [hunk 2]
```diff
@@ -623,11 +623,11 @@ <h1 id="form">Form Example</h1>
 
 {% block left_sidebar %}
 <aside class="left-sidebar" role="secondary">
-  
+
     {% include "components/navigation-widget.html" %}
 
     <div class="twitter-widget sidebar-widget">
-        <script type="text/javascript" src="http://widgets.twimg.com/j/2/widget.js"></script>
+        <script type="text/javascript" src="https://widgets.twimg.com/j/2/widget.js"></script>
```
Comment (berkerpeksag) [inline templates/python/inner.html]: It would better to use the same style in both this file and `templates/waitforit.html` (`//...` was used instead of `https://...` there)

File: templates/python/inner.html [hunk 3]
```diff
@@ -623,11 +623,11 @@ <h1 id="form">Form Example</h1>
 
 {% block left_sidebar %}
 <aside class="left-sidebar" role="secondary">
-  
+
```
Comment (berkerpeksag) [inline templates/python/inner.html]: Unrelated change.

File: pages/tests/fake_svn_content_checkout/about/success/dlink/content.rst [hunk 4]
```diff
@@ -8,7 +8,7 @@ D-Link Australia Uses Python to Control Firmware Updates
 :Author: How Wong
 :Contact: grandsonata at aol dot com
 :Date: 2007/08/02 10:00:00
-:Web site: http://www.dlink.com.au/
+:Web site: https://www.dlink.com.au/
```
Comment (berkerpeksag) [inline pages/tests/fake_svn_content_checkout/about/success/dlink/content.rst]: This is used by the test suite, no need to change.

File: pages/tests/fake_svn_content_checkout/about/content.ht [hunk 5]
```diff
@@ -31,29 +31,29 @@ Python is powerful... and fast
 ------------------------------
 
 Fans of Python use the phrase "batteries included" to describe the
-`standard library <http://docs.python.org/3/library/>`_, which covers
+`standard library <https://docs.python.org/3/library/>`_, which covers
```
Comment (berkerpeksag) [inline pages/tests/fake_svn_content_checkout/about/content.ht]: This file is used by the test suite, no need to change.

File: codesamples/factories.py [hunk 6]
```diff
@@ -42,7 +42,7 @@ def initial_data():
             <p>Experienced programmers in any other language can pick up Python very
             quickly, and beginners find the clean syntax and indentation structure
             easy to learn.
-            <a href=\"http://docs.python.org/3.3/tutorial/\">Whet your appetite</a>
+            <a href=\"https://docs.python.org/3/tutorial/\">Whet your appetite</a>
```
Comment (berkerpeksag) [inline codesamples/factories.py:45]: This file creates data for development purposes. No need to change.

File: blogs/tests/test_models.py [hunk 7]
```diff
@@ -9,7 +9,7 @@ class BlogModelTest(TestCase):
     def test_translation(self):
         t = Translation.objects.create(
             name='Swiss',
-            url='http://python.ch',
+            url='https://python.ch',
```
Comment (berkerpeksag) [inline blogs/tests/test_models.py]: This is just a test, no need to change.

File: blogs/tests/test_models.py [hunk 8]
```diff
@@ -21,7 +21,7 @@ def test_blog_entry(self):
             title='Test Entry',
             summary='Test Summary',
             pub_date=now,
-            url='http://www.revsys.com',
+            url='https://www.revsys.com',
```
Comment (berkerpeksag) [inline blogs/tests/test_models.py]: This is just a test, no need to change.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

File: users/tests/test_models.py (hunk 1 above)
Comment (JulienPalard) [reply users/tests/test_models.py]: I agree, I however though it would be good to have some https in tests just in case it break something. But it should better be separated tests in this case so I revert this one OK.

File: codesamples/factories.py (hunk 6 above)
Comment (JulienPalard) [reply codesamples/factories.py:45]: Why not having up-to-date links in dev too?

Comment (berkerpeksag) [reply codesamples/factories.py:45]: The test data in codesamples/factories.py is basically for frontend developers to see the slider in action, but sure let's update it.

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks.
